source/features/latest-tag-button.tsx

```diff
@@ -131,7 +131,7 @@ async function init(): Promise<false | void> {
 		link.setAttribute(
 			'aria-label',
 			isAhead
-				? `${defaultBranch} is ${pluralize(aheadBy, '1 commit', '$$ commits')} ahead of the latest version`
+				? `${defaultBranch} is ${aheadBy === undefined ? 'more than 20 commits' : pluralize(aheadBy, '1 commit', '$$ commits')} ahead of the latest version`
```

Comment:
Should we also put adapt what's displayed in that case after the `+` in the tag button? Currently there's no number at all, which might be confusing.

Reply:
How about an asterisk? Or <sup>`+?`</sup> or <sup>`+🤷‍♂️`</sup>

Reply:
Let's go with the asterisk. We could also use `++` instead if it's not visible enough.